cmd/restic/global.go

```diff
@@ -105,6 +107,13 @@ func init() {
 		return nil
 	})
 
+	//set MinPackSize to 4 if not set in env
+	minPackSize, err := strconv.Atoi(os.Getenv("RESTIC_MIN_PACKSIZE"))
```

Comment:
If you use strconv.ParseUint, you don't need the unsignInt hack below. Also, the comment is incorrect.

Reply:
Oh, good to know.

Also, good catch on the comment, thanks.